README.md

```diff
@@ -12,14 +12,7 @@ Be sure to install [VS Code](https://code.visualstudio.com/Download) and create
 
```

Comment:
Prerequisites should say:

1. Create a [FaunaDB account](https://dashboard.fauna.com/accounts/register).
2. Install [VS Code](https://code.visualstudio.com/Download).
3. Install this extension either from [within VS Code](https://code.visualstudio.com/docs/editor/extension-gallery), or from [a vsix file](https://github.com/eamodio/vscode-gitlens/wiki/Installing-Prereleases-(vsix)).

Reply:
I would just ask to remove the `from a vsix file` we don't want to enforce that. We should always enforce to install trough the vscode store so the user can get automatic updates.

Reply:
Can you please send a commit with these changes? I don't know for sure what should be the best format for this topic.